ground-station-openmct/terminal/terminal_logic.js

```diff
@@ -1,18 +1,56 @@
 const terminal = document.getElementById("terminal");
 const input = document.getElementById("input");
 const form = document.getElementById("terminalForm");
+let output = null
+let output_pk = 1
+
+setInterval(function() {
+    fetch(`http://127.0.0.1:8000/data/terminalOutput/${output_pk}/`)
+    .then(response => response.json())
+    .then(data => {
+        console.log(data)
+        output = data.terminal_output;
+    })
+    .catch(error => {
+      console.error('Failed to fetch data from endpoint:', error);
+    })
+
+    console.log(output)
+    if (output) {
+        terminal.innerHTML += `$ ${output}\n`;
+        output_pk++;
+    }
+}, 3000);
+
 
 form.addEventListener("submit", function (event) {
     event.preventDefault();
-    const command = input.value;
-    input.value = "";
+    const fullCommand = input.value;
+    let ack = fullCommand.match(/(\d+)/)[0];
+    ack = parseInt(ack)
+    const command = fullCommand.replace(/ .*/,'');
 
-    terminal.innerHTML += `$ ${command}\n`;
+    input.value = "";
 
     if (command === "clear") {
       terminal.innerHTML = "";
-    } else if (command === "cutdown") {
-      terminal.innerHTML += "success\n";
+    } else if (command == "cutdown" || command == "ping") {
```

Comment:
what if instead of parsing the command in the frontend you just send the whole thing to the backend?  then it's less places to have to change when we add a new command type